packages/bpk-component-rating/README.md

```diff
@@ -31,12 +31,15 @@ export default () => (
 | title     | string                | true     | -                 |
 | value     | number                | true     | -                 |
 | className | string                | false    | null              |
+| pill      | boolean               | false    | false             |
```

Comment:
An enum is preferable for styling properties, as it leaves the API open for any future styles that need to be added. 

Consider if in future we have to add a 'square' type. If we use a boolean we'd be stuck, but if we have a `RATING_TYPE` enum we can just add a new type.